src/ducks/files/Toolbar.jsx

```diff
@@ -29,6 +29,8 @@ class Toolbar extends React.Component {
 
   render () {
     const { t, disabled, displayedFolder, actions, onSelectItemsClick, uploadFiles } = this.props
+    const notRootfolder = displayedFolder && displayedFolder.path !== '/'
+    console.log(displayedFolder);
```

Comment:
There's still a `console.log` here ⚠️
And so it's not possible to use the root folder id to do this test?